airbyte-integrations/connectors/source-plaid/integration_tests/catalog.json

```
@@ -105,6 +106,12 @@
             "check_number": { "type": ["string", "null"] },
             "datetime": { "type": ["string", "null"] },
             "merchant_name": { "type": ["string", "null"] },
+            "name": { "type": ["string"] },
```

For a single type, you don't need an array.

```suggestion
            "name": { "type": "string" },
```

This is also true for the identical code in `configured_catalog.json`.